pkg/trait/knative_service.go

```diff
@@ -175,6 +178,8 @@ func (t *knativeServiceTrait) getServiceFor(e *Environment) (*serving.Service, e
 		for k, v := range e.Integration.Annotations {
 			serviceAnnotations[k] = v
```

Comment:
It seems the current behavior to set the annotations in the integration and they are propagated to the knative service is actually right, isn't ?
If the user sets the knative service route annotation, then the integration annotations are not propagated to the knative service, which is not backward compatible.

Reply:
Yes. We need to append the trait annotation after the Integration annotations are propagated.